MUXSDKStats/Podfile

```diff
@@ -5,12 +5,14 @@ target 'MUXSDKStats' do
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
   # use_frameworks!
   # Pods for MUXSDKStats
-  pod 'Mux-Stats-Core', '~>3.3'
+#  pod 'Mux-Stats-Core', '~>3.3'
+  pod 'Mux-Stats-Core', :git => 'https://github.com/muxinc/stats-sdk-objc.git', :branch => 'bmulligan/customerdata-without-customerviewerdata'
```

Comment:
I just merged `bmulligan/customerdata-without-customerviewerdata` so this should be able to be updated to `:branch => 'release'`